package.json

```diff
@@ -10,6 +10,7 @@
     "migrate:undo": "sequelize db:migrate:undo:all",
     "seed": "sequelize db:seed:all",
     "start:dev": "npm run migrate && nodemon server/app.js --exec babel-node",
+    "test": "export NODE_ENV=test || SET \"NODE_ENV=test\" && nyc mocha tests/**/*.test.js --timeout 20000 --require @babel/register --exit",
```

Comment:
What do you need the `SET` command for?

Reply:
Coming from a windows OS background I normally would have that on my tests script to enable me to set my test env. export works for mac/linux

Reply:
Since we are working with mac/linux environments (both locally and for testing) you can remove this command

Reply:
Will do.